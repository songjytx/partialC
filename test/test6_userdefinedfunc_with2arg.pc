float testcall(int b, float a){
	b = b + 1;
	printi(b);
	if(b > 5){
		return 0.0;
	}else{
		testcall(b, 1.9);
	}
	printf(a);
	return -1.9;
}

void main(){
	int i=0;
	
	testcall(0, 0.0);
}