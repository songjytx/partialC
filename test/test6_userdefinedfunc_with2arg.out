1
2
3
4
5
6
1.900000
1.900000
1.900000
1.900000
0.000000
